tests/unittests/Foundation/NSStringInternalTests.mm

```diff
@@ -14,6 +14,8 @@
 //
 //******************************************************************************
 
+#ifdef _WIN32
```

Comment:
same, though that might say more about our test design? other comments seem to indicate that a lot of these can go away by me using shims instead of ifdefs

---
In reply to: [70349454](https://github.com/Microsoft/WinObjC/pull/634#discussion_r70349454) [](ancestors = 70349454)